docs/source/create/entities/zone-entity.rst

```diff
@@ -0,0 +1,164 @@
+##########################
+Add a Zone Entity
+##########################
+
+A zone entity allows you to set properties to a specific area. These properties include: flying, ghosting, key lighting, skybox, ambient lighting, hase, bloom and avatar priorities.
```

Comment:
```suggestion
A Zone entity enables you to set properties in a specific volume. These properties include: flying, ghosting, key lighting, skybox, ambient lighting, haze, bloom, and avatar priorities.
```

Reply:
Even though "volume" is more specific that "area", I feel like could would confuse people. Especially non-native english speakers.

Reply:
Fair enough.